pkg/update/update.go

```diff
@@ -42,6 +46,28 @@ func ParsePackage(pkg string) (name, version string) {
 	return nameWithVersion[0], version
 }
 
+// GetDateBasedVersion returns the latest date version for the given version.
+func GetDateBasedVersion(v *buildsafev1.FetchPackagesResponse, version string) string {
+	if v == nil {
+		return ""
+	}
+
+	var latestVersion string
+	var latestDate string
+
+	for _, pkg := range v.Packages {
+		if strings.HasPrefix(pkg.Version, version[:4]) {
```

Comment:
There is a pkg.EpochSeconds field . We need to use that instead of `pkg.Version`.

Reply:
something like this `string(pkg.EpochSeconds)` ?